tools/appveyor/install.ps1

```diff
@@ -3,31 +3,10 @@ $ErrorActionPreference = "Stop"
 try {
     & git submodule --quiet update --init --recursive
 
-    Write-Host -ForegroundColor Green "`n### Installing CMake and python ###`n"
-    & cinst --no-progress cmake python2
```

Comment:
We need both python versions, python2 and python3 since we are testing both. Not sure if they are already preinstalled

Reply:
Yes, they're both installed